infra/frontend/app-config/env-config/environment-variables.tf

```diff
@@ -44,6 +44,11 @@ locals {
       manage_method     = "manual"
       secret_store_name = "/${var.app_name}/${var.environment}/api-auth-token"
     },
+    # URL for the API login route. 
+    AUTH_LOGIN_URL = {
```

Comment:
This isn't secret so it goes next to the NODE_OPTIONS key above

Reply:
It is the same as the `API_URL` right?

Reply:
@coilysiren Secrets knows how to read from SSM where as the standard env vars are more "constants?" Right? Or is there a way to read from SSM there too?